xbmc/addons/AddonManager.cpp

```diff
@@ -364,9 +364,17 @@ bool CAddonMgr::GetAddonsInternal(const TYPE& type,
   return addons.size() > 0;
 }
 
-bool CAddonMgr::GetIncompatibleAddons(std::vector<AddonInfoPtr>& incompatible) const
+bool CAddonMgr::GetIncompatibleEnabledAddons(std::vector<AddonInfoPtr>& incompatible) const
+{
+  return GetIncompatibleAddons(incompatible, false);
+}
```

Comment:
This is OK, only maybe not good to have the `GetIncompatibleAddons` also public then where outside can call this `GetIncompatibleAddons(incompatible, false)` too.

Then maybe better to leave away or make the `GetIncompatibleAddons` private and add another function with
```cpp
bool CAddonMgr::GetIncompatibleAllAddons(std::vector<AddonInfoPtr>& incompatible) const
{
  return GetIncompatibleAddons(incompatible, true);
}
```

Reply:
Done, went with making it private.